spec/factories/author.rb

```diff
@@ -54,6 +54,32 @@
     end
   end
 
+  factory :period_first_name_author, parent: :author do
```

Comment:
Have you considered using traits for this?

Reply:
just learned about this, adjusted to use